public/js/dec-bin-hex-converter

```diff
@@ -0,0 +1,47 @@
+(function(){
+
+    var ConvertBase = function (num) {
```

Comment:
why did not you make to , from as parameters for this function. instead of this complicated implementation

`function (num ,from , to)`
`return  parseInt(num, baseFrom).toString(baseTo);`